lte/gateway/c/core/oai/tasks/nas5g/src/M5GIdentityResponse.cpp

```diff
@@ -74,8 +65,8 @@
 }
 
 // Encode Identity Response Message and its IEs
-int IdentityResponseMsg::EncodeIdentityResponseMsg(
-    IdentityResponseMsg* identity_response, uint8_t* buffer, uint32_t len) {
+int IdentityResponseMsg::EncodeIdentityResponseMsg(uint8_t* buffer,
+                                                   uint32_t len) {
```

Comment:
No change in indentation has been made, only the removal of unnecessary field to avoid misuse of method and ease the comprehension of the code.
The indentation is automatically placed in the devcontainer environment.